cdi/3.0/_index.md

```diff
@@ -1,26 +1,30 @@
 ---
-title: "Jakarta Context Dependency Injection 3.0 (milestone)"
-date: 2020-05-28
+title: "Jakarta Context Dependency Injection 3.0"
+date: 2020-08-06
 summary: "Release for Jakarta EE 9"
 ---
 
 Jakarta Contexts Dependency Injection specifies a means for obtaining objects in such a way as to maximize reusability, testability and maintainability compared to traditional approaches such as constructors, factories, and service locators (e.g., JNDI).
 
 * [Jakarta Contexts Dependency Injection 3.0 Release Record](https://projects.eclipse.org/projects/ee4j.cdi/releases/3.0)
   * [Jakarta EE Platform 9 Release Plan](https://eclipse-ee4j.github.io/jakartaee-platform/jakartaee9/JakartaEE9ReleasePlan)
-* [Jakarta Contexts Dependency Injection 3.0 Specification Document](./cdi-spec-3.0.0-M4.pdf) (PDF)
-* [Jakarta Contexts Dependency Injection 3.0 Specification Document](./cdi-spec-3.0.0-M4.html) (HTML)
+* [Jakarta Contexts Dependency Injection 3.0 Specification Document](cdi-spec-3.0.pdf) (PDF)
+* [Jakarta Contexts Dependency Injection 3.0 Specification Document](cdi-spec-3.0.html) (HTML)
```

Comment:
Did you want to use the "jakarta-" prefix for the spec documents?

Reply:
Done